feat(konnect-platform): Additional audit log info (#6469)
Kong/developer.konghq.com: app/konnect-platform/audit-logs.md

    dev_portal_audit_logs: Triggered when a user logs in to the {{site.dev_portal}}.
    name_value: "`AUTHENTICATION_OUTCOME_*`"
  - event_type: Authorization
    org_audit_logs: "Triggered when a permission check is made for a user or system account against a resource."
    dev_portal_audit_logs: Not collected, use the org audit log.
    name_value: "`Authz.*`"
  - event_type: Access logs
    org_audit_logs: "Triggered when a request is made to the {{site.konnect_short_name}} API."
    dev_portal_audit_logs: Not collected, use the org audit log.
    name_value: "`Ingress`"
{% endtable %}
<!--vale on-->

{{site.konnect_short_name}} retains audit logs for 7 days. After 7 days, {{site.konnect_short_name}} permanently deletes them and you cannot recover them.

{:.info}
> * {{site.dev_portal}} audit logs don't collect authorization and access events by design. You can view {{site.dev_portal}} entity creation, edits, and approved state changes from the {{site.konnect_short_name}} audit logs. 

> > * Don't rely on `event_class_id` to identify the log type. Its meaning differs per event type, and for authorization logs it isn't a fixed enum, it's the name of whichever Kong platform service performed the authorization check.

Reviewer: ```suggestion
> * Don't rely on `event_class_id` to identify the log type. Its meaning differs per event type.
> For authorization logs, it's the name of whichever Kong platform service performed the authorization check, not a fixed enum value.
```
For clarity/breaking up the run-on.

Authorization log entries are created for every permission check in {{site.konnect_short_name}}.

The `name` field always starts with `Authz.` followed by the resource path that was checked:

<!--vale off-->
{% table %}
columns:
  - title: "`name` pattern"
    key: name_pattern
  - title: When
    key: when
rows:
  - name_pattern: "`Authz.organization`"
    when: Organization-level operation
  - name_pattern: "`Authz.control-planes/*`"
    when: Wildcard across all control planes
  - name_pattern: "`Authz.control-planes/{id}`"
    when: Specific control plane

>   - name_pattern: "`Authz.api-products/{id}`"

Reviewer: These are just examples, right? If that's the case, I'd remove API Product, since they're supposed to be deprecated.

>     when: Specific API Product

Author: ```suggestion
```
